### DISCUSSION TITLE
Standard keyboard brightness keys don't work on macOS 12.5

### DISCUSSION THREAD

**Discussion by mejedi** [MonitorControl/MonitorControl#1125] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

After update to macOS 12.5, standard keyboard brightness keys stopped working.

### Steps to reproduce

1. Go to app preferences, Keyboard.
2. Have at least one external monitor attached.
3. Ensure that `Brightness and contrast` is set to `Both standard and custom shortcuts`.
4. When using custom shortcuts, external monitor brightness changes as expected.
5. Standard keys only change brightness on laptop monitor, as if MonitorControl didn't receive the keypress.

### Expected behavior

Standard keyboard brightness keys should work as they used to before the update.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.5
- Mac model:  MacBook Pro (16-inch, 2019)
- MonitorControl version: 4.1.0
- Monitor(s): LG HDR 4K
- Apple Silicon/M1 (yes or no): no
```

**Comment by NchemIcaLS** [comment]
I have the same issue. It works if I use custom keys but not if I use the standard keys.

- macOS version: 12.4
- Mac model:  MacBook Pro (16-inch, 2016)
- MonitorControl version: 4.1.0
- Monitor(s): Gigabyte M32U
- Apple Silicon/M1 (yes or no): no

**Comment by waydabber** [comment]
It might be possible that you need to re-add the application to the Security/Accessibility list so keyboard shortcuts start working again. Otherwise the feature should work under 12.5 as well.

**Reply by mejedi** [reply]
Great news! It works now. I have checked Security/Accessibility, the app was there and the checkbox was set. Having cleared the checkbox and set it again fixed the issue for me.

**Reply by NchemIcaLS** [reply]
In my case the app wasn't listed after opening it for the first time yesterday. But this also fixed it for me.

**Comment by waydabber** [comment]
(btw. only the native keys require the Accessibility permission to work, custom shortcuts do not)

**Comment by NchemIcaLS** [comment]
Explicitly adding the app to the accessibility permission worked! Also need to restart the app for permission to take effect.

I installed the app for the first time yesterday. When I launched it the first time it asked me about the accessibility permission but strangely the feature already had a green checkmark next to it and I couldn't click it. I also checked system preferences > security & privacy after but it didn't ask to allow anything. Maybe there is a bug in the initial check.

**Comment by waydabber** [comment]
Generally this feature in macOS works only semi-well to be honest and is a pain to deal with. :) An app sometimes get listed and seems to be enabled while it is actually not, often times simply because it was opened from an other location in the past or some signature difference or various other factors. So it is always a good practice to try to re-add the app or if all else fails, simply reset the accessibility permissions list.

**Comment by alfonsocv12** [comment]
This is broken again in 13.5 I guess 14 has the same issue. I try what @waydabber suggested with no luck.

Currently monitorControll has access to Accessibility and input monitoring in the Privacy & Security section.

**Reply by waydabber** [reply]
You might need to reset accessibility permissions and re-add the app.

https://www.macworld.com/article/347452/how-to-fix-macos-accessibility-permission-when-an-app-cant-be-enabled.html

You should exit all apps beforehand as this command sometimes causes the system to be unresponsive to keyboard and mouse inputs until a sleep cycle or sometimes even a restart is needed.
